src/lib/components/query/QueryNav.svelte

```diff
@@ -1,15 +1,13 @@
 <script lang="ts">
 	import { Add, Arrow_right, Menu } from "svelte-google-materialdesign-icons";
 	import { queries } from "$lib/globalState/activeProject";
-	import { Backend } from "$lib/classes/automaton/Backend";
+	//import { Backend } from "$lib/classes/automaton/Backend";
```

Comment:
I honestly think it is better to keep it. It is only temporarily disabled because it will be re-added by a different PR. So we are keeping it here to show the old implementation until the new implementation is introduced. 

If you still think it is better to remove, I can do that as well.